pkg/rtc/transport.go

```diff
@@ -702,6 +703,19 @@ func (t *PCTransport) SetPreferTCP(preferTCP bool) {
 }
 
 func (t *PCTransport) AddICECandidate(candidate webrtc.ICECandidateInit) {
+	if !t.params.Config.UseMDNS {
+		candidateValue := strings.TrimPrefix(candidate.Candidate, "candidate:")
+		if candidateValue != "" {
+			candidate, err := ice.UnmarshalCandidate(candidateValue)
+			if err != nil {
+				t.params.Logger.Errorw("failed to parse ice candidate", err)
+			} else if strings.HasSuffix(candidate.Address(), ".local") {
+				t.params.Logger.Debugw("ignoring mDNS candidate", "candidate", candidateValue)
+				return
+			}
+		}
+	}
```

Comment:
moving this in here from transport manager as this has correct logging context and easier to parse logs.